server/fishtest/templates/tests_view.mak

```diff
@@ -860,3 +860,27 @@
     % endif
     });
 </script>
+<script>
+    document.getElementById('gamesDownloadButton').addEventListener('click', function() {
```

Comment:
Nice! one issue is that this will cause a run-time if the button doesn't exist (i.e. approver is false).. in such cases we use `?.addEventListener` in the code base
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
```js
const response = await fetch('/api/run_pgns/${run['_id']}.pgn.gz');
```
I wonder if the use of the quote is consistent with our codebase.

Reply:
I don't think so, prettier default is double quotes.